Update tutorial in README (#20)
bdlukaa/system_theme: README.md

### Get system accent color

Use the getter `SystemTheme.accentColor.accent` to get the system accent color.

```dart
final accentColor = SystemTheme.accentColor.accent;
```

To reload the accent colors, use the method `load()`:

```dart
await SystemTheme.accentColor.load();
```

You can load the colors before running the app, so the colors can't be wrong at runtime:

```dart
void main() async {

>   WidgetsFlutterBinding.ensureInitialized();

Reviewer: ```suggestion
```

The `SystemAccentColor` already invoke the method `ensureInitialized`, meaning it needless to execute it beforehand.

Author: I'm not sure why `SystemAccentColor` invoke that method since it should be invoked at user-developer's code. For example, [firebase/flutterfire never call that in their lib code, but in their example code.](https://github.com/search?q=repo%3Afirebase%2Fflutterfire%20WidgetsFlutterBinding.ensureInitialized()&type=code)

So here may be another issue. This suggestion will not be accepted.

  await SystemTheme.accentColor.load();
  runApp(MyApp());
}
```

### Configure a fallback accent color

> To avoid unexpected effects at runtime, it's good to configure your own fallback color. The fallback color is used if the system was not able to provide the color.

Reviewer: ```suggestion
To avoid unexpected outcomes at runtime, it's recommended to configure your own fallback color. The fallback color is used if the system is not able to provide the color.
```

```dart
void main() async {

>   WidgetsFlutterBinding.ensureInitialized();
> 
>   SystemTheme.fallbackColor = const Color(0xFF865432);
>   await SystemTheme.accentColor.load();

Reviewer: It's tested that the code above make NO SENSE as the assignment of fallback color have NO EFFECT on the accent color...

```suggestion
  var fallbackColor = const Color(0xFF865432);
  SystemTheme.accentColor = await SystemAccentColor(fallbackColor)..load();
```

Author: No, it does affect the accent color but only when platform channel call returns `null`.

![image](https://github.com/bdlukaa/system_theme/assets/18349191/d2cd3980-bfd2-4b5f-ab2f-5e47378088e5)

Reviewer: > No, it does affect the accent color but only when platform channel call returns `null`.
> 
> ![image](https://user-images.githubusercontent.com/18349191/244549595-d2cd3980-bfd2-4b5f-ab2f-5e47378088e5.png)

Uhhhh...
I didn't try in that way...I tried first load it, change the fallbackColor and load it again.
The fallbackColor setter works only when it's the first to be called in the whole runtime, meaning that the code below is invalid

```dart
//void main() {
// Retrieve user-stored color preference
// maybe some io or database conn
// will callback to set systemTheme fallback color later in that part
// Try to load color first
SystemTheme.accentColor.load()
// runApp()
}
```

And if the accentColor is not loaded(like in iOS), when the user change the color preference, the systemTheme.fallbackColor is changed, but it doesn't affect the systemTheme.accentColor, so developers have to define some separate code for it...

Anyway, present in the package are holes of the old-era, i have chosen to write one myself in my app yesterday...
